src/utils.ts

```diff
@@ -6,12 +6,25 @@ export function useUnmount(fn: () => void) {
     useEffect(() => fn, EMPTY_ARRAY)
 }
 
+let skippingForceUpdate = 0
```

Comment:
Ugh, that doesn't sound right to be relying on a global variable...For multi-threading environments, it could cause some problems.

Reply:
Luckily then Js is not multithreaded really :)

Reply:
Well, I heard that NextJS somehow is :)